### PR TITLE
Edit to topics/edge: remove locale from url

### PR DESCRIPTION
**Pull request by mikaeldui**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Content (and my changes are in `index.md`)

> The current URL has the locale "en-us", but there's a generic URL without locale which might be better suited, and looks cleaner.

### CODE REVIEW

File: topics/edge/index.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ logo: edge.png
...
 short_description: Edge is a cross-platform web browser created and developed by Microsoft.
 topic: edge
-url: https://www.microsoft.com/en-us/edge
+url: https://www.microsoft.com/edge
 wikipedia_url: https://en.wikipedia.org/wiki/Microsoft_Edge
 ---
 
-Microsoft Edge is a cross-platform web browser created and developed by Microsoft. First released for Windows 10 in 2015, for Android and iOS in 2017, for macOS in 2019, and for Linux in 2020, can replace Internet Explorer on Windows 7, Windows Server 2008 R2 and later versions.
\ No newline at end of file
+Microsoft Edge is a cross-platform web browser created and developed by Microsoft. First released for Windows 10 in 2015, for Android and iOS in 2017, for macOS in 2019, and for Linux in 2020, can replace Internet Explorer on Windows 7, Windows Server 2008 R2 and later versions.
```
Comment (kenyonj) [inline topics/edge/index.md:14]:
```suggestion
Microsoft Edge is a cross-platform web browser created and developed by Microsoft. First released for Windows 10 in 2015, for Android and iOS in 2017, for macOS in 2019, and for Linux in 2020, can replace Internet Explorer on Windows 7, Windows Server 2008 R2 and later versions.
```

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: topics/edge/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikaeldui) [reply topics/edge/index.md:14]: Will try my best Git magic.

[the remark this answers is not in the document]
Comment (mikaeldui) [reply topics/edge/index.md]: That _change_ has now been removed.

**Review by kenyonj — APPROVED** [review]

**Pull request merged**